ci/cpu/build.sh

```diff
@@ -88,8 +88,8 @@ if [ "$BUILD_CUSPATIAL" == '1' ]; then
   if [[ -z "$PROJECT_FLASH" || "$PROJECT_FLASH" == "0" ]]; then
     gpuci_conda_retry build --croot ${CONDA_BLD_DIR} conda/recipes/cuspatial
   else
-    gpuci_conda_retry build --croot ${CONDA_BLD_DIR} --dirty --no-remove-work-dir \
-        -c $WORKSPACE/ci/artifacts/cuspatial/cpu/.conda-bld/ conda/recipes/cuspatial
+    gpuci_conda_retry build --croot "$CONDA_BLD_DIR" --dirty --no-remove-work-dir \
+        -c "$WORKSPACE/ci/artifacts/cuspatial/cpu/conda-bld/" conda/recipes/cuspatial
```

Comment:
```suggestion
        -c "$WORKSPACE/ci/artifacts/cuspatial/cpu/.conda-bld/" conda/recipes/cuspatial
```

looks like a `.` was mistakenly removed here.